packages/calcite-components/src/components/inline-editable/inline-editable.scss

```diff
@@ -1,36 +1,37 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-inline-editable-background-color: Specifies the background color of the component.
+ * @prop --calcite-inline-editable-size: Specifies the size of the component.
```

Comment:
Should this be public? I feel like this kind of conflates with `scale`. If it is public, should it be `height` as it is applied to `block-size` ?

Reply:
Great catch. 🎣 Will fix.